package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-pattern-library",
-  "version": "0.13.0",
+  "version": "0.13.1",
```

Comment:
Note that we update the version number only in a release PR. See https://openedx.atlassian.net/wiki/display/FEDX/UI+Toolkit+release+process